plugins/linkattacher/src/linkattacher.cc

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2013-2015 Fondazione Istituto Italiano di Tecnologia RBCS & iCub Facility & ADVR
+ * Authors: see AUTHORS file.
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or any later version, see LGPL.TXT or LGPL3.TXT
+ */
+
+#include <linkattacher.hh>
+
+using namespace gazebo;
+using namespace std;
+using namespace yarp::os;
+
+LinkAttacher::LinkAttacher() : ModelPlugin(), m_network(), m_rpcport()
+{
+}
+
+LinkAttacher::~LinkAttacher()
+{
+  if(m_rpcport)
+  {
+    m_rpcport->close();
+  }
+}
+
+void LinkAttacher::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf)
+{
+  if(m_network!=0)
+  {
+    return;
+  }
+
+  m_network = std::unique_ptr<yarp::os::Network> (new yarp::os::Network());
```

Comment:
Minor comment: no space between <> and ()

Reply:
Done